seatunnel-core/seatunnel-starter/src/main/bin/seatunnel-cluster.sh

```diff
@@ -69,7 +70,8 @@ do
   if [[ "${i}" == *"JvmOption"* ]]; then
     JVM_OPTION="${i}"
     JAVA_OPTS="${JAVA_OPTS} ${JVM_OPTION#*=}"
-    break
+  elif [[ "${i}" == "-d" ]]; then
```

Comment:
Please make sure if user use `-h` or `--help`, the help msg must print on the console. At now if user use `./bin/seatunnel-cluster.sh -h -d`, the console will not print anything. By the way, user use `-h` can't know how to use daemon mode (please add it into `CommandArgs`)